### PR TITLE
push code for new project

### PR DESCRIPTION
**Pull request by uyenpham-viedoc**

### CODE REVIEW

File: UyenProject/Lesson3_Calculate.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            {
+                Console.WriteLine("Division of 2 numbers:" + (num1 / num2));
+            }
+            else if (num2 == 0)
+
+            {
+                Console.WriteLine("Can not divide to 0");
+            }
+        }
+
```
Comment (HuongDoViedoc) [inline UyenProject/Lesson3_Calculate.cs:61]: I am not sure whether we need to add 4 methods for operators here or just calculate in calculator method ? Waiting for feedback from Mr Trung.

File: UyenProject/Lesson3_Validate.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+            while (!arrOperations.Contains(operation))
+            {
+                Console.WriteLine(" Wrong!!Enter operator again  +  -  *  / : ");
+                operation = Console.ReadLine();
+            }
+            return operation;
+        }
+    }
+}
```
Comment (HuongDoViedoc) [inline UyenProject/Lesson3_Validate.cs:34]: After reading about Naming test method convention, I think we should rename the method here. ex: Validate_and_Return_CorrectInputInteger and Validate_and_Return_CorrectInputOperater

File: UyenProject/Program.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        Lesson3_Calculate.Calculator(operation, num1, num2);
+
+        //Lesson4
+
+        //string fullstring = Lesson4.readAndcheckSubstring();
+        //string substring = Lesson4.readAndcheckSubstring();
+
+    }
+
+}
```
Comment (HuongDoViedoc) [inline UyenProject/Program.cs:22]: With Net6, we do not need to call methods in Main method of Class Program. Remove unused code for cleaning and clear code :))

File: UyenProject/Lesson3_Validate.cs (hunk 2 above)
Comment (viedoc-trungle) [reply UyenProject/Lesson3_Validate.cs:34]: No, it's about test methods, not methods in our program.

File: UyenProject/Lesson3_Calculate.cs (hunk 1 above)
Comment (viedoc-trungle) [reply UyenProject/Lesson3_Calculate.cs:61]: I think we can go either way. But if the method has only 1 line of code, maybe we don't have the separate it.

**Review by viedoc-trungle — APPROVED** [review]

**Pull request merged**